core/domain/exp_jobs_one_off_test.py

```diff
@@ -752,3 +752,95 @@ def test_that_mapreduce_job_works(self):
+                <b>Hello Oppia</b> message</pre>. <p> Don't you want to
+                say hello? You can learn more about oppia
+                <oppia-noninteractive-link url-with-value="&amp;quot;
+                https://www.example.com&amp;quot;" text-with-value="&amp;quot;
+                here&amp;quot;"></oppia-noninteractive-link></p>''',
+            'audio_translations': {}
+        }
+        content2_dict = {
+            'html': '''<pre>Hello, this is state2.</pre> <blockquote>
+                <ol> <li>item1</li> <li>item2</li> </ol> </blockquote>.<p>
+                You can see this equation <b> <oppia-noninteractive-math
+                raw_latex-with-value="&amp;quot;\\frac{x}{y}&amp;
+                quot;"></oppia-noninteractive-math></b></p>
+                ''',
+            'audio_translations': {}
+        }
+        state1.update_content(content1_dict)
+        state2.update_content(content2_dict)
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        # Start validation job on sample exploration.
+        job_id = exp_jobs_one_off.ExplorationContentValidationJob.create_new()
+        exp_jobs_one_off.ExplorationContentValidationJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        actual_output = (
+            exp_jobs_one_off.ExplorationContentValidationJob.get_output(job_id))
+        expected_output = [u"[u'Errors', [u'{}']]"]
+
+        self.assertEqual(actual_output, expected_output)
+
+        default_outcome_dict = {
+            'dest': 'State2',
+            'feedback': {
+                'html': '''<p>Sorry, it doesn't look like your <span>program
+                </span>prints any output</p>.<p> <blockquote> Could you get
+                it to print something?</blockquote> You can do this by using a
+                statement like print</p>. <br> You can ask any doubt if you have
+                <oppia-noninteractive-link url-with-value="&amp;quot;
+                https://www.example.com&amp;quot;" text-with-value="
+                &amp;quot;Here&amp;quot;"></oppia-noninteractive-link>.
+                ''',
+                'audio_translations': {}
+            },
+            'labelled_as_correct': False,
+            'param_changes': [],
+            'refresher_exploration_id': None
+        }
+
+        state1.update_interaction_default_outcome(default_outcome_dict)
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        job_id = exp_jobs_one_off.ExplorationContentValidationJob.create_new()
+        exp_jobs_one_off.ExplorationContentValidationJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        actual_output = (
+            exp_jobs_one_off.ExplorationContentValidationJob.get_output(job_id))
+        expected_output = [u'[u\'Errors\', [u"{\'invalidTags\': [u\'span\'], u\'oppia-noninteractive-link\': [u\'[document]\'], u\'br\': [u\'[document]\']}"]]'] # pylint: disable=line-too-long
```

Comment:
Break the string up:

    expected_output = [
        '...'
        '...'
        '...'
    ]

Try to avoid adding pragmas if you can avoid it.

Reply:
Done in #4969